README.md

```diff
@@ -11,44 +11,7 @@ This repository contains the common JavaScript functions used by Node.js collect
 
 # HOWTO consume this library in Node.js
 
-Create a `package.json` file for [npm](https://www.npmjs.com/) 2.7.0 (or greater) 
-in the root of your Node.js directory.  Include this repo in the 
-`dependencies` and `devDependencies` section as required.  
-
-For example:
-
-```
-{
-  "name": "al-my-collector",
-  "version": "1.0.0",
-  "description": "Alert Logic My Collector",
-  "repository": {},
-  "private": true,
-  "scripts": {
-    "start": "node index.js",
-    "lint": "jshint --exclude \"./node_modules/*\" **/*.js",
-    "test": "JUNIT_REPORT_PATH=./test/report.xml nyc --reporter=cobertura mocha --colors --reporter mocha-jenkins-reporter"
-  },
-  "devDependencies": {
-    "dotenv": "*",
-    "clone": "*",
-    "jshint": "^2.9.5",
-    "mocha": "^3.5.3",
-    "mocha-jenkins-reporter": "^0.3.10",
-    "nyc": "^11.3.0",
-    "rewire": "^2.5.2",
-    "sinon": "^3.3.0"
-  },
-  "dependencies": {
-    "async": "*",
-    "moment": "^2.19.2",
-    "request": "*",
-    "request-promise-native": "*",
-    "al-collector-js": "git://github.com/alertlogic/al-collector-js#master"
-  },
-  "author": "Alert Logic Inc."
-}
-```
+`npm install --save @alertlogic/al-collector-js`
```

Comment:
could you please ad some code sample how to consume the lib API, like
https://github.com/alertlogic/al-azure-collector-js/blob/master/collector.js#L58
https://github.com/alertlogic/al-azure-collector-js/blob/master/collector.js#L60
https://github.com/alertlogic/al-azure-collector-js/blob/master/collector.js#L104